distributions/demo-server/build.gradle

```diff
@@ -197,24 +195,24 @@ task copyWebApp {
     }
 }
 
-task buildClientImage(type: DockerBuildImage) {
-    description = 'Builds the Client Docker image'
-    dependsOn shadowJar, createClientDockerfile, copyWebApp
-    dockerFile = project.file("${buildDir}/client/Dockerfile")
+task buildDemoImage(type: DockerBuildImage) {
+    description = 'Builds the Demo Docker image including API server and nginx'
+    dependsOn shadowJar, createDemoDockerfile, copyWebApp
+    dockerFile = project.file("${buildDir}/demo/Dockerfile")
     inputDir = project.file(".")
-    tag = 'dataportability/client:latest'
+    tag = 'datatransferproject/demo:latest'
 }
 
-task dockerize {
-    description = 'Builds the Client and API Server Docker images'
+task demoserver {
```

Comment:
can demoserver and buildDemoImage be combined? if so I would keep the convention of naming the task dockerize, since we're in the demo distro, and the task we are doing is creating a docker image of the whole thing